README.md

```diff
@@ -1 +1,34 @@
 # GRID-FRAMEWORK
+## The project to design custome made Grid based framework 
+using
+* SASS
+ #I used to clone the 
+ [Ask media website ](https://raw.githack.com/IBUNHABIBU/GRID-FRAMEWORK/gridframework/index.html)
```

Comment:
`HINT`

Add link the source page for comparison. [Source Page](https://www.askmediagroup.com/)`(https://www.askmediagroup.com/)`